package/package.json

```diff
@@ -29,6 +29,7 @@
   },
   "dependencies": {
     "@material-ui/core": "^3.1.0",
+    "fs-extra": "^7.0.0",
```

Comment:
`fs-extra` isn't needed as a package dep. Did you intend to update the one that is a dev dep in the other package.json?

Reply:
I added `fs-extra` because I was getting errors that it is required but not found. `.reaction/scripts/addcomponent.js` has `const fs = require("fs-extra");` on line 1. Without this change, if `fs-extra` is ever imported, in the situations where it "works" it is because another packages happens to have already installed `fs-extra` and we are therefore relying on an accident.

The other change in this file, `reacto-form`, is just `npm` putting the packages in alphabetical order.

Reply:
Oh I see what you mean. `fs-extra` is a dev-dependency in the root `package.json` folder. I will remove adding it from this PR.